Container: Typescript constructor definition is missing

### Please tell us about your environment:
  
- _`winston` version?_
   - [ ] `winston@2`
   - [x] `winston@3` 
- _Language?:_ TypeScript

### What is the problem? 
In the current Typescript definitions there is no `new`/`constructor` definition for the `Container` class. This leads to a compiler error, when you try to instantiate a new custom Container in Typescript. 

`new winston.Container(options)` --> "error TS2351: Cannot use 'new' with an expression whose type lacks a call or construct signature."

### What do you expect to happen instead?
Please add the missing definition to the `index.d.ts`.

--- a/index.d.ts
+++ b/index.d.ts
@@ -128,13 +128,15 @@ declare namespace winston {
   }
 
   interface Container {
-    loggers: object;
-    options: object;
+    loggers: Map<string, Logger>;
+    options: LoggerOptions;
 
     add(id: string, options?: LoggerOptions): Logger;
     get(id: string, options?: LoggerOptions): Logger;
     has(id: string): boolean;
-    close(): void;
+    close(id?: string): void;
+
+    new(options?: LoggerOptions): Container;
   }
 
   let version: string;

--- a/test/typescript-definitions.ts
+++ b/test/typescript-definitions.ts
@@ -1,6 +1,6 @@
 import * as winston from '../index';
 
-const logger: winston.Logger = winston.createLogger({
+let logger: winston.Logger = winston.createLogger({
     level: 'info',
     format: winston.format.json(),
     transports: [
@@ -26,7 +26,26 @@ const loggerOptions: winston.LoggerOptions = {
         new winston.transports.Console({ level: 'info' }),
     ],
 };
-winston.loggers.add('category', loggerOptions);
-winston.loggers.get('category', loggerOptions);
-winston.loggers.has('category');
+
+// assign the returned values to the logger variable,
+// to make sure that the methods have 'Logger' declared as their return type
+logger = winston.loggers.add('category', loggerOptions);
+logger = winston.loggers.add('category');
+logger = winston.loggers.get('category', loggerOptions);
+logger = winston.loggers.get('category');
+
+const hasLogger: boolean = winston.loggers.has('category');
+winston.loggers.close('category');
 winston.loggers.close();
+
+let container: winston.Container = new winston.Container(loggerOptions);
+logger = container.get('testLogger');
+
+logger = container.loggers.get('testLogger')!;
+
+container.close('testLogger');
+
+const level = container.options.level;
+
+container = new winston.Container();
+logger = container.get('testLogger2');

